torchao/quantization/quant_primitives.py

```diff
@@ -2270,6 +2270,7 @@ def _expand_scale_to_tensor_shape(
     return expanded_scale
 
 
+@_register_custom_op(quant_lib, False)
```

Comment:
we can probably make `quantize_affine_float8` and `choose_qparams_affine_float8` public as well since it's used in inductor lowering. cc @jainapurva @drisspg